src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SentenceAdjustedCalculationService.kt

```diff
@@ -248,6 +248,7 @@ class SentenceAdjustedCalculationService(
   private fun getBotusTusedDate(sentence: BotusSentence, sentenceCalculation: SentenceCalculation) {
     if (sentence.latestTusedDate?.isAfter(sentenceCalculation.releaseDate) == true) {
       sentenceCalculation.topUpSupervisionDate = sentence.latestTusedDate
+      sentenceCalculation.breakdownByReleaseDateType[TUSED] = tusedCalculator.getCalculationBreakdownForBotus(sentenceCalculation)
```

Comment:
I think the method where this code sits would be better placed in the TusedCalculator class + there are other bits in here that do TUSED specific calc/breakdown stuff that I think should be moved. 

I didnt do it as part of this PR though, will discuss and perhaps do as a tech debt ticket